qvm-tools/qvm-terminal-dispvm

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+print_usage() {
+cat >&2 << USAGE
+Usage: $0 vmname
+Connects to VM console throught DispVM using the admin.vm.TerminalDispVM RPC service.
+USAGE
+}
+
+if [ $# -lt 1 ]; then
+    print_usage
+    exit 1
+fi
+
+QREXEC_REQUESTED_TARGET="$1"
+
+qvm-check --quiet "$QREXEC_REQUESTED_TARGET" > /dev/null 2>&1 || { echo "Error: no such domain: '$QREXEC_REQUESTED_TARGET'"; exit 1; }
+
+sudo qvm-run -p --localcmd="QREXEC_REQUESTED_TARGET=$QREXEC_REQUESTED_TARGET /etc/qubes-rpc/admin.vm.Terminal" --service --dispvm="$(qubes-prefs management_dispvm)" -- admin.vm.TerminalDispVM
```

Comment:
Retrieve `management_dispvm` separately and check if it isn't empty. 
Use VM's property, not the global one.
`admin.vm.TerminalDispVM` of VM service is bad - `admin.*` are services to control qubes configuration ("Admin API"). This service merely show in terminal whatever you sent on its stdin. Also, the service itself is equally useful in non-DispVM, so including DispVM in its name is misleading. Maybe `qubes.ShowInTerminal`?